### DISCUSSION TITLE
Flickering when reloading

### DISCUSSION THREAD

**Discussion by danilo-leal** [vercel/next.js#20246] [Help] [answered]
Hey guys.
I'm a bit bugged by the flickering at my website when I reload the page. Does anyone have any idea why that is?
👉 [daniloleal.co](daniloleal.co)

Thanks a lot for helping out!

**Comment by leerob** [comment, marked as answer]
What you're witnessing is how web fonts work, specifically Flash of Unstyled Text (FOUT). 

There are many alternatives to consider, depending on which makes the most sense for your site. 

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display
